pywrap/test/test_libclang.py

```diff
@@ -1,7 +1,7 @@
-from pywrap.libclang import find_clang
+from pywrap.libclang import find_clang, SUPPORTED_VERSIONS
 from nose.tools import assert_in
 
 
 def test_find_clang():
     CLANG_VERSION, CLANG_INCDIR = find_clang(set_library_path=False, verbose=2)
-    assert_in(CLANG_VERSION, ["3.5", "3.6", "3.7", "3.8"])
\ No newline at end of file
+    assert_in(CLANG_VERSION, SUPPORTED_VERSIONS)
```

Comment:
This is not totally correct any more because only the major and minor version is returned.